src/DssSpell.sol

```diff
@@ -60,222 +58,192 @@ contract DssSpellAction is DssAction {
+            dstMaxLine: 7_500 * MILLION * RAD,
 
-        // JuliaChang - 1.25 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 1.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Note: Increase by 180 million DAI, from 20 million DAI to 200 million DAI.
+            dstGap:     200 * MILLION * RAD,
 
-        // Rocky - 1.13 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 1.13 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Note: LITE-PSM-USDC-A DC-IAM ttl: 12h (Unchanged)
+            dstTtl:     12 hours,
 
-        // PBG - 1.00 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        MKR.transfer(PBG, 1.00 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Note: chainlog key of old psm
+            srcPsmKey:  "MCD_PSM_USDC_A",
 
-        // ---------- Aligned Delegate DAI Compensation ----------
-        // Forum: https://forum.makerdao.com/t/july-2024-aligned-delegate-payment-requests/24794
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-budget-and-participation-requirements
+            // Note: PSM-USDC-A tin: Increase by 0.01 percentage points, from 0% to 0.01%
+            srcTin:     0.0001 ether, // Note: ether is a keyword helper, no transfers are made here
 
-        // BLUE - 54167 DAI - 0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf
-        DssExecLib.sendPaymentFromSurplusBuffer(BLUE, 54_167);
+            // Note: PSM-USDC-A tout: Increase by 0.01 percentage points, from 0% to 0.01%
+            srcTout:    0.0001 ether, // Note: ether is a keyword helper, no transfers are made here
 
-        // Cloaky - 20417 DAI - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        DssExecLib.sendPaymentFromSurplusBuffer(CLOAKY, 20_417);
+            // Note: PSM-USDC-A DC-IAM DC-IAM line: Decrease by 7,500 million DAI, from 10,000 million DAI to 2,500 million DAI.
+            srcMaxLine: 2_500 * MILLION * RAD,
 
-        // Kohla (Cloaky) [NEW ADDRESS] - 14172 DAI - 0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a
-        DssExecLib.sendPaymentFromSurplusBuffer(CLOAKY_KOHLA_2, 14_172);
+            // Note: PSM-USDC-A DC-IAM DC-IAM gap: Decrease by 180 million DAI, from 380 million DAI to 200 million DAI.
+            srcGap:     200 * MILLION * RAD,
 
-        // Ennoia (Cloaky) - 9083 DAI - 0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b
-        DssExecLib.sendPaymentFromSurplusBuffer(CLOAKY_ENNOIA, 9_083);
+            // Note: PSM-USDC-A DC-IAM DC-IAM ttl: 12h (Unchanged)
+            srcTtl:     12 hours,
 
-        // Byteron - 8333 DAI - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        DssExecLib.sendPaymentFromSurplusBuffer(BYTERON, 8_333);
+            // Note: Migrate all but 200 million USDC reserves from PSM-USDC-A to LITE-PSM-USDC-A
```

Comment:
`Note:` prefix is required when a) there is no matching instruction in the sheet b) you don't follow the instruction that is specified above. See [the checklist](https://github.com/makerdao/pe-checklists/blob/e2376d43fe55e606669c48b44c41b0b8e7de70cc/spell/spell-crafter-mainnet-workflow.md?plain=1#L74-L75). In this particulat case: [matching instruction exists in the Exec Sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=175785542#gid=175785542&range=B43), so it doesn't need `Note:` prefix

Reply:
fixed in [dc2a705](https://github.com/makerdao/spells-mainnet/pull/422/commits/dc2a705f34094bd4850410f13a12afa27c11a6f1)